src/replication.c

```diff
@@ -500,6 +499,59 @@ trigger_rebuild(spec_t *spec)
 	}
 }
 
+static bool
+is_replica_familiar(spec_t *spec, replica_t *new_replica)
+{
+	known_replica_t *kr = NULL;
+	int familiar_replicas = 0;
+	bool ret = false, found = false;
```

Comment:
its better to change the name of 'ret' to 'newly_connected'

Reply:
ok